diff --git a/host/cmake/Modules/UHDBoost.cmake b/host/cmake/Modules/UHDBoost.cmake
index eed94cbfe..54d1a837f 100644
--- a/host/cmake/Modules/UHDBoost.cmake
+++ b/host/cmake/Modules/UHDBoost.cmake
@@ -94,15 +94,6 @@ if(UHD_BOOST_OPTIONAL_COMPONENTS_LEN EQUAL 0 AND
     return()
 endif()
 
-# if the OS is MINGW and if 'thread' is in the list, change its name
-if(MINGW)
-    list(FIND UHD_BOOST_REQUIRED_COMPONENTS "thread" THREAD_NDX)
-    if(NOT ${THREAD_NDX} EQUAL -1)
-        list(REMOVE_AT UHD_BOOST_REQUIRED_COMPONENTS ${THREAD_NDX})
-        list(INSERT UHD_BOOST_REQUIRED_COMPONENTS ${THREAD_NDX} thread_win32)
-    endif()
-endif()
-
 # if 'system' is in the list, make sure it comes last; sometimes this
 # linkage makes a difference
 list(FIND UHD_BOOST_REQUIRED_COMPONENTS "system" SYSTEM_NDX)
